charts/argocd-image-updater/templates/configmap-sshconfig.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
I know we use this structure in this repository but it looks a bit strange. WDYT about:

```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  labels:
    {{- include "argocd-image-updater.labels" . | nindent 4 }}
  name: argocd-image-updater-ssh-config
data:
  {{- with .Values.config.sshConfig }}
  {{- toYaml . | nindent 2 }}
  {{- end }}
```

In this chart theres already another configmap which implement a feature in this structure: `configmap-authscripts.yaml`

Reply:
I also prefer to drop the data in the values.yaml. It felt also a bit strange to me, i copied that from another chart and thought that would be the prefered approach.

I will change that to your suggestion.